routes/store.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
You can just just use `express.json()` instead of importing `body-parser`, unless we need to use the form-encoding parser